### ISSUE TITLE
Docker for Windows throwing error

### ISSUE THREAD

**Issue by kuhlaid** [Colectica/cogs#308]
Tried to setup Docker environment from http://cogsdata.org/docs/quick-start/docker-quick-start/ using:
```s cmd
docker volume create sandbox
docker run -v sandbox:/sandbox colectica/cogs cogs-new /sandbox/MyModel
```
Wound up with a `No executable found matching command "dotnet-/app/Cogs.Console/out/Cogs.Console.dll"` error.

**Comment by borsna** [comment]
I think the Docker-file needs some work.
Using the official base image with specific .net version should work.
So changing `Dockerfile` to:

```Dockerfile
FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env

WORKDIR /App

# Copy everything
COPY . ./

# Restore as distinct layers
RUN dotnet restore ./Cogs.Console.sln
# Build and publish a release
RUN dotnet build ./Cogs.Console.sln -c Release -o out

# Build runtime image
FROM mcr.microsoft.com/dotnet/aspnet:6.0
WORKDIR /App
COPY --from=build-env /App/out .

ENTRYPOINT ["dotnet", "cogs.dll"]
```
works for me

**Comment by DanSmith** [comment]
fixed by @borsna in #315
